blocks/term-tree-select/index.js

```diff
@@ -18,7 +18,7 @@ function getSelectOptions( terms, level = 0 ) {
 	] );
 }
 
-export default function TermTreeSelect( { termsTree, label, noOptionLabel, selectedTerm, onChange } ) {
+export default function TreeSelect( { termsTree, label, noOptionLabel, selectedTerm, onChange } ) {
```

Comment:
Should the `termsTree` prop be renamed to something more generic?

Reply:
Sorry, I was un the middle of the process, it was updated with this and other reactors. And readme file was added.